src/cells/model.ts

```diff
@@ -276,10 +306,21 @@ class BaseCellModel implements IBaseCellModel {
   type: CellType;
 
   /**
-   * Re-emit changes to the input dirty state.
+   * Handle changes to the input model.
+   */
+  protected onInputChanged(input: IInputAreaModel, args: IChangedArgs<any>): void {
+    // Re-emit changes to input dirty and readOnly states.
+    if (args.name === 'dirty' || args.name === 'readOnly' || args.name ) {
```

Comment:
Won't this re-emit *every* input area change now (because of the `|| args.name`)? Is that intentional?